src/hearings/containers/cancel-hearing/cancel-hearing.component.ts

```diff
@@ -1,27 +1,56 @@
 import { Component, OnInit } from '@angular/core';
-import { FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { FormArray, FormBuilder, FormGroup } from '@angular/forms';
 import { ActivatedRoute } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { ACTION, CancelHearingMessages } from 'src/hearings/models/hearings.enum';
+import { HearingListModel } from '../../../hearings/models/hearingList.model';
 import { RefDataModel } from '../../../hearings/models/refData.model';
 import { HearingsService } from '../../../hearings/services/hearings.service';
-
+import * as fromHearingStore from '../../store';
+import { ValidatorsUtils } from '../../utils/validators.utils';
+import { RequestHearingPageFlow } from '../request-hearing/request-hearing.page.flow';
 @Component({
   selector: 'exui-cancel-hearing',
   templateUrl: './cancel-hearing.component.html',
   styleUrls: ['./cancel-hearing.component.scss']
 })
-export class CancelHearingComponent implements OnInit {
+export class CancelHearingComponent extends RequestHearingPageFlow implements OnInit {
```

Comment:
This shouldn't use the pageflow, as it's a simple single page. This will overkill the page flow to handle unnecessary page logic.